src/routes/student/Shop.tsx

```diff
@@ -51,11 +53,19 @@ export default function Shop({ player, classroom }: { player: Player; classroom:
 		setValue(newValue)
 	}
 
+	const divStyle = {
```

Comment:
I don't want to leave too many comments but see if there's a way to do all of your `divStyle` components in MUI instead